packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/nlb/network-load-balancer.ts

```diff
@@ -465,6 +475,13 @@ export interface INetworkLoadBalancer extends ILoadBalancerV2, ec2.IVpcEndpointS
    */
   readonly ipAddressType?: IpAddressType;
 
+  /**
+   * Indicates whether to evaluate inbound security group rules for traffic sent to a Network Load Balancer through AWS PrivateLink
+   *
+   * @default on
+   */
+  readonly enforceSecurityGroupInboundRulesOnPrivateLinkTraffic?: EnforceSecurityGroupInboundRulesOnPrivateLinkTraffic;
```

Comment:
```suggestion
   * @default true
   */
  readonly enforceSecurityGroupInboundRulesOnPrivateLinkTraffic?: boolean;
```
What about defining it as a boolean? It seems a bit overkill to define an enum for representing `on`/`off` values

Reply:
That would make way too much sense. I'll make the change 👍🏼😂